src/SwedbankPay.Sdk/PaymentInstruments/MobilePay/IMobilePayPaymentsResource.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments.MobilePayPayments
+{
+    public interface IMobilePayPaymentsResource
+    {
+        Task<IMobilePayPayment> Create(MobilePayPaymentRequest paymentRequest, PaymentExpand paymentExpand = PaymentExpand.None);
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)